sops/european-level/GDI-SOP0009_dataset-withdrawal.md

```diff
@@ -0,0 +1,466 @@
+|:------------------|:----|:----|
+| ``4``             |After requester authority is verified|VHD Officer|
+
+Given that the request is valid and the requester has authority to withdraw the dataset:
+- **Create a VHD ticket** to tackle the dataset withdrawal itself:
+   - Create it with **type** ``dataset-withdrawal``.
+   - **Link it** to the initial ``dataset-withdrawal-request`` VHD ticket that was generated with the requester communication. As per your system, this link should imply that the ``dataset-withdrawal-request`` ticket "is blocked by" this new ``dataset-withdrawal`` ticket.
+   - Flag its ``priority`` as ``High``.
+- **Escalate the ticket**. Use the following email template to **communicate the request** with **your direct manager** (in CC) within GDI, the OC (``gdi-oc [at] elixir-europe.org``) and SDPC (``gdi-sdpc [at] elixir-europe.org``). _Note the ``[at]`` to be replaced with ``@``_.
+   - Include relevant stakeholders from within GDI (not the requester!) in CC.
+   - Attach to the email any **supporting documentation** (e.g., verification of authority, screenshots, initial request...).
+````
+To: gdi-sdpc [at] elixir-europe.org, gdi-oc [at] elixir-europe.org
+CC: <your-direct-manager>
+Subject: [<dataset-withdrawal TICKET_ID>][GDI SOP0009] Dataset withdrawal request
+````
+````
+Dear OC/SDPC,
+
+A dataset-withdrawal request has been verified and logged in the GDI Virtual Helpdesk.
+
+Request details:
+- Ticket ID: <Internal TICKET_ID from GDI's VHD> (<TICKET_URL if applicable>)
+- Date/time received: <DATETIME as YYYY.MM.DD>
+- Requester contact details:
+   - <REQUESTER_NAME> (<REQUESTER_ROLE>, <REQUESTER_INSTITUTION>)
+   - <REQUESTER_EMAIL_ADDRESS>
+- Dataset IDs: <list of relevant dataset IDs>
+- Scope: <Whether the dataset is to be fully or partially withdrawn>
+- Stated reason: <Reason for withdrawal from the requester>
+- Requester authority: <The results of your investigation at step 3 of this SOP>
+- Deadline requested: <Deadline of withdrawal if applicable>
+
+ACTIONS NEEDED [Within 5 working days]:
+1. Please designate technical lead(s) to:
+   a. Identify all GDI systems where this dataset (or the specified subset) resides and is to be withdrawn from.  
+   b. Notify each system owner and coordinate the withdrawal schedule.  
+2. Update the VHD ticket with assigned leads and expected completion timeline.
+
+See more details at [Step 5](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/sops/european-level/GDI-SOP0009_dataset-withdrawal.md#85-assign-withdrawal-response-team).
+
+Once all systems confirm completion, the VHD will resume control to verify, notify the requester, and close the ticket.
+
+Thank you for your prompt attention.  
+
+Kind regards,  
+<YOUR_NAME>
+<YOUR_ROLE>
+<YOUR_INSTITUTION>
+GDI Virtual Helpdesk
+````
+
+- **Send acknowledgement to requester** within 5 working days. Use the following template, including ticket ID and summary of next steps.
```

Comment:
As the ticket ID will be an internal "operations" ticket, this information does not need to be shared with the user. As well, the answer should be sent in the original request ticket the user created to request the withdrawal since this is a follow-up.